### ISSUE TITLE
Accessibility Issues that Should be Fixed

### ISSUE THREAD

**Issue by nlottig94** [nlottig94/Dickinson#11]
# Overall Changes
- [x] Add an `xml:lang` attribute on all html pages.
- [x] Change the color of the side navigation buttons or the text, not enough contrast
- [x] Mark up the above mentioned navigation bar as a list

# Homepage
- [x] Mark up navigation at bottom of page to GitHub pages as a list OR change it to just have the _current_ GitHub.

# Bibliography
- [ ] Create a `listBibl` for this page instead of just coding it as paragraphs?
